clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-16-vexxhost-hybrid_clusterpool.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    version: "4.16"
+    version_lower: 4.16.0-0
+    version_upper: 4.17.0-0
+  name: oko-op-07
+    namespace: openstack-k8s-operators-cluster-pool
+spec:
+  baseDomain: oooci.ccitredhat.com
+  hibernationConfig: null
+  imageSetRef:
+    name: ocp-release-4.16.0-ec.6-x86-64-for-4.16.0-0-to-4.17.0-0
```

This is not the image we would like to use. The `-ec` means engineering candidate, more details here [1]

We want to use [2], this is the moving GA release of 4.16"

```suggestion
    name: ocp-release-4.16.3-x86-64-for-4.16.0-0-to-4.17.0-0
```

[1] https://mirror.openshift.com/pub/openshift-v4/OpenShift_Release_Types.pdf
[2] https://github.com/openshift/release/blob/master/clusters/hosted-mgmt/hive/pools/ocp-release-4.16.3-x86-64-for-4.16.0-0-to-4.17.0-0_clusterimageset.yaml

---

Done